src/pages/app/posting_project_user/styles/ProfilePage.css

```diff
@@ -86,16 +85,15 @@
   /*margin*/
   margin-bottom: 5%;
   /*padding*/
-  padding: 20px;
-  padding-left: 5%;
+  padding: 25px;
+  padding-left: 5.8437%;
```

Comment:
Please use **even** numbers

Reply:
I calculated it based the value on Figma. This precision was to make the page looks as close as Figma as possible.